keras/utils/timeseries_dataset.py

```diff
@@ -128,6 +130,23 @@ def timeseries_dataset_from_array(
   X = np.arange(100)
   Y = X*2
 
+  sample_length = 20
+  dataset = tf.keras.preprocessing.timeseries_dataset_from_array(
+    X, Y, sequence_length=sample_length, sequence_stride=sample_length, seq_to_seq=True)
```

Comment:
when user have seq_to_seq == False in this case, what will be the output here?

Reply:
This use case is logically only if seq_to_seq = True